cfme/infrastructure/virtual_machines.py

```diff
@@ -564,9 +565,11 @@ def active(self):
                 bool: True if snapshot is active, False otherwise.
             """
             self._nav_to_snapshot_mgmt()
+            title = self.description if self.vm.provider.one_of(RHEVMProvider) else self.name
+            self._click_tree_path(title)
```

Comment:
This is a duplicate of the call that's inside the try block, think you want to remove this one.

Reply:
You're right, missed that.